includes/core/server/request/class-get-account.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Class file for WCPay\Core\Server\Request\Get_Account.
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core\Server\Request;
+
+use WCPay\Core\Server\Request;
+use WC_Payments_API_Client;
+
+/**
+ * Request class for getting intents.
+ */
+class Get_Account extends Request {
```

Comment:
@Automattic/gamma I'd like to have early feedback regarding to this trait. We're having 4 requests like this under the current class `WC_Payments_API_Client`.

Reply:
Marking as resolved, and moving this convo to https://github.com/Automattic/woocommerce-payments/pull/5879#discussion_r1155454194